test/appium/tests/atomic/chats/test_public.py

```diff
@@ -297,6 +299,42 @@ def test_public_navigate_to_chat_when_relaunch(self):
         if not self.chat.chat_element_by_text(text_message).is_element_displayed(30):
             self.drivers[0].fail("Not navigated to chat view after reopening app")
 
+    @marks.testrail_id(5317)
+    @marks.critical
+    def test_public_copy_and_paste_messages(self):
+        message_text = {'text_message': 'mmmeowesage_text'}
+        formatted_message = {'message_with_link': 'https://status.im',
```

Comment:
flaky test in general; as sometimes message as a button can't be tapped (happens from time to time, tried to tap by coordinates, but didn't help). Perhaps it is better  to move to medium, WDYT?

Reply:
Agree, moved it to a group together with `test_different_status_in_timeline`, probably we can add `test_gap_in_public_chat_and_no_gap_in_1_1_and_group_chats` there too?